changelogs/fragments/win_dsc-write-error-standardize.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- win_dsc - deduplicated error writing code with a new function. # no error text was changed just exit logic de-duplicated
```

Comment:
Just as an FYI the comment here will no appear in the changelog, it will be lost when the changelog itself is generated on the next release.